Makefile

```diff
@@ -56,7 +56,7 @@ else
 
     # Resolve all symbols at link time
     ifeq ($(SYS),Linux)
-    	CFLAGS += -DX11_GAMMA_WORKAROUND
+        CFLAGS += -DX11_GAMMA_WORKAROUND -fno-tree-vectorize
```

Comment:
Please keep this in a separate commit. Not sure we should merge it tbh. What you could do it keep a local .config file where you add it to your CFLAGS for builds with higher optimization levels. (O3 is rarely recommended due to many potential bugs, is there a reason to why you want to compile with O3?)